subprojects/docs/src/docs/userguide/depMngmt.adoc

```diff
@@ -496,6 +496,9 @@ There are several different types of repositories you can declare:
 | <<sub:maven_jcenter,Maven JCenter repository>>
 | A pre-configured repository that looks for dependencies in Bintray's JCenter.
 
+| <<sub:maven_google,Maven Google repository>>
+| A pre-configured repository that looks for dependencies in Google repository.
```

Comment:
Above we say `Bintray's` repository not `Bintray JCenter`  Google repository is weird.  Going with @big-guy suggestion above maybe we should consider `Google's Maven repository`?